File: Julia/Snowflake/Connection.jl
using Snowflake


function conectar()
    conn = SnowflakeConnection(
        account = "SEU ACCOUNT",
        warehouse = "SEU WAREHOUSE",
        db = "SEU DATABASE",
        user = "SEU USER",
        password = "SEU PASSWORD",
    )

    conn.open()

    return conn
end


function execute_query(conn, query)
    result = conn.execute(query)

    return result
end


function desconectar(conn)
    conn.close()
end

File: Ruby/Snowflake/Connection.rb
require "snowflake"


def conectar()
    conn = Snowflake::Connection.new(
        account: "SEU ACCOUNT",
        warehouse: "SEU WAREHOUSE",
        db: "SEU DATABASE",
        user: "SEU USER",
        password: "SEU PASSWORD",
    )

    conn.open

    return conn
end


def execute_query(conn, query)
    result = conn.execute(query)

    return result
end


def desconectar(conn)
    conn.close
end